velox/exec/tests/WindowTest.cpp

```diff
@@ -487,5 +487,41 @@ TEST_F(WindowTest, nagativeFrameArg) {
   }
 }
 
+DEBUG_ONLY_TEST_F(WindowTest, frameColumnNullCheck) {
```

Comment:
Curious why we make it DEBUG_ONLY test?

Reply:
@kagamiori : Because the assertion which crashed was a DCHECK. It didn't really participate in any logic which had correct offsets. If we had commented out the DCHECK, the code would've worked correctly.

I feel that code should retain as a DCHECK only. We don't anticipate it at runtime, but is an assumption about how things are setup.